lib/string_reverse.rb

```diff
@@ -1,6 +1,16 @@
 # A method to reverse a string in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the number of characters in string
+# Space complexity: O(1)
 def string_reverse(my_string)
-  raise NotImplementedError
+  return nil if my_string == nil

```

Comment:
🤔 Since `" "` and `nil` are not the same, would you want to go through the process of reversing a string that is empty?